dziq/sharingan.cfg

```diff
@@ -0,0 +1,5 @@
+setshader bumpspecdecal
+setshaderparam bumpblend 0.8
+setshaderparam specscale 0.2
+    texture decal "decals/dziq/sharingan"
```

Comment:
Isn't this a copyrighted emblem?

Reply:
Yeah, it is, from naruto.

Reply:
yeah I rushed with this,  hard to find any info so I removed it